developer-docs-site/scripts/additional_dict.txt

```diff
@@ -582,3 +583,5 @@ ANSClient
 FaucetClient
 CoinClient
 RESTful
+Transafer
```

Comment:
Let's, not add `Transafer` to the dictionary, and instead of course use the word `Transfer`